web/db/migrations/20260806020000_coderouter_hosted_entitlements/migration.sql

```
@@ -0,0 +1,7 @@
+-- Expand first. The application rejects unscoped rows after deployment; a
+-- follow-up migration removes them and makes this column NOT NULL.
+ALTER TABLE "coderouter_route_tokens"
+  ADD COLUMN IF NOT EXISTS "stack_user_id" text;
+
+CREATE INDEX IF NOT EXISTS "coderouter_route_tokens_user_expiry_idx"
+  ON "coderouter_route_tokens" ("stack_user_id", "expires_at");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)